test-tools/core/src/main/java/org/kie/cloud/tests/core/resources/Project.java

```diff
@@ -0,0 +1,335 @@
+            OpenShiftBinary oc = openShiftBinaryClient();
+            if (openShift.getServiceAccount("apb") == null) {
+                oc.execute("create", "serviceaccount", "apb");
+                oc.execute("create", "rolebinding", "apb", "--clusterrole=admin", "--serviceaccount=" + projectName + ":apb");
+            }
+
+            List<String> args = new ArrayList<>();
+            args.add("run");
+            args.add(podName);
+            args.add("--env=POD_NAME=" + podName);
+            args.add("--env=POD_NAMESPACE=" + projectName);
+            args.add("--image=" + image);
+            args.add("--restart=Never");
+            args.add("--attach=true");
+            args.add("--serviceaccount=apb");
+            args.add("--");
+            args.add("provision");
+            args.add("--extra-vars");
+            args.add(formatExtraVars(extraVars));
+
+            LOGGER.info("Executing command: oc {}", getApbCommand(args));
+            oc.execute(args.toArray(new String[0]));
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+            throw new RuntimeException("Interrupted while waiting for scenario to be initialized.", e);
+        } finally {
+            semaphore.release();
+        }
+    }
+
+    private String getApbCommand(List<String> args) {
+        return args.stream().collect(Collectors.joining(" "));
+    }
+
+    private String formatExtraVars(Map<String, String> extraVars) {
+        return extraVars.entrySet()
+                .stream()
+                .map(entry -> "\"" + entry.getKey() + "\":\"" + entry.getValue() + "\"")
+                .collect(Collectors.joining(", ", "{", "}"));
+    }
+
+    /**
+     * Create all resources defined in resource URL.
+     *
+     * @param resourceUrl URL of resource list to be created
+     */
+    public void createResources(String resourceUrl) {
+        try {
+            KubernetesList resourceList = openShift.lists().inNamespace(projectName).load(new URL(resourceUrl)).get();
+            openShift.lists().inNamespace(projectName).create(resourceList);
+        } catch (MalformedURLException e) {
+            throw new RuntimeException("Malformed resource URL", e);
+        }
+    }
+
+    /**
+     * Create resources from YAML file using command line client.
+     * @param yamlUrl Url to yaml file with resources
+     */
+    public void createResourcesFromYaml(String yamlUrl) {
+        final String output = openShiftBinaryClient().execute("create", "-f", yamlUrl);
+        LOGGER.info("Yaml resources from file {} were created by oc client. Output = {}", yamlUrl, output);
+    }
+
+    /**
+     * Create resources from YAML files using command line client.
+     * @param yamlUrls Urls to yaml files with resources
+     */
+    public void createResourcesFromYaml(List<String> yamlUrls) {
+        final OpenShiftBinary oc = openShiftBinaryClient();
+        for (String url : yamlUrls) {
```

Comment:
will be better two replace this method with just calling of createResourcesFromYaml(String) method. We will follow clean code principles and also we can be sure that if change will done for first method, it's not needed and forgot to update the second method.
e.g. `yamlUrls.forEach(url -> createResourcesFromYaml(url));`

Reply:
done :)